tests/foreman/cli/test_syncplan.py

```diff
@@ -272,33 +188,20 @@ def test_positive_update_2(self, test_data):
         @Assert: Sync plan interval is updated
 
         """
-        new_sync_plan = self._make_sync_plan({
-            u'interval': test_data['interval'],
-            u'name': test_data['name'],
-        })
-        # Assert that name and interval matches data passed
-        self.assertEqual(new_sync_plan['name'], test_data['name'])
-        self.assertEqual(new_sync_plan['interval'], test_data['interval'])
-        # Update sync interval
-        SyncPlan.update({
-            u'id': new_sync_plan['id'],
-            u'interval': test_data['new-interval'],
-        })
-        # Fetch it
-        result = SyncPlan.info({u'id': new_sync_plan['id']})
-        # Assert that interval was updated
-        self.assertEqual(result['interval'], test_data['new-interval'])
-        self.assertNotEqual(new_sync_plan['interval'], result['interval'])
-
-    @data(
-        {u'name': gen_string('alpha', 15)},
-        {u'name': gen_string('alphanumeric', 15)},
-        {u'name': gen_string('numeric', 15)},
-        {u'name': gen_string('latin1', 15)},
-        {u'name': gen_string('utf8', 15)},
-        {u'name': gen_string('html', 15)},
-    )
-    def test_positive_update_3(self, test_data):
+        for test_data in valid_name_interval_update_tests():
+            with self.subTest(test_data):
+                new_sync_plan = self._make_sync_plan({
+                    u'interval': test_data['interval'],
+                    u'name': test_data['name'],
+                })
+                SyncPlan.update({
+                    u'id': new_sync_plan['id'],
+                    u'interval': test_data['new-interval'],
+                })
+                result = SyncPlan.info({u'id': new_sync_plan['id']})
+                self.assertEqual(result['interval'], test_data['new-interval'])
+
+    def test_positive_update_3(self):
```

Comment:
Earlier this was a ddt test for no reason.  All this test tries to do is to update sync data.  So I created a single test to do just that.